script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -391,67 +418,75 @@ That's so... sad.
 [sha:07eea14e3babd84451283f9273a391e9c0591340]{
 -- Page 412, Offset 34461.
 -- こんな時に、それが嘘だと分かってしまうのは、皮肉すぎる。
-At a time like this, I'm finally able to see through her lies. How ironic.
+Only now that things have come to this am I able to see through her lies...
+ The irony is crushing.
 }
 [sha:b7b35791f18e1a1a2752c305d04e05a1c88e995e]{
 -- Page 413, Offset 34462.
 -- 「……ひどいな。最後まで、先輩は俺に嘘をつくのか」
-"...How awful. Do you intend on lying to me until the very end, Senpai?"
+"I can't believe it...
+ Do you really plan on lying to me right up until the very end, Senpai?"
 }
 [sha:f0dd4a6a183c39fe5dad0df55f71db7a3d60ca20]{
 -- Page 413, Offset 34463.
 -- 「―――――――」
 "―――――――"
 }
-[sha:68a938c0eaed172db6b8953cba8d5faa7e319dae]{
+[offset:34464]{
 -- Page 413, Offset 34464.
 -- 返答はない。
-There's no answer.
+She doesn't answer.
 }
 [sha:f417c9d90a852cbaa37487186c9fa1669775b646]{
 -- Page 413, Offset 34465.
 -- 凍りついたように、先輩は動かない。
-Senpai is completely frozen in place.
+Senpai is motionless, as though frozen in place.
 }
 [sha:2822fb04328a9dff68430a13b7bad1f62c3b8160]{
 -- Page 414, Offset 34466.
 -- 「―――貴方こそ、嘘をついてる。ここでわたしに殺されることが望みなんて、思ってはいないでしょう」
-"―――The same could be said for you. You don't truly wish to die by my hands, do you?"
+"―――I could say the same to you.
+ You don't actually want to die here by my hand, do you?"
 }
 [sha:c862719f578dfa33067b244238ea14e06c307a6e]{
 -- Page 414, Offset 34467.
 -- 「……当然だろ、そんなの。死んだら何もないじゃないか。一応経験があるから、それぐらいは分かってる。
-"...Of course not. If I die, then that'll be the end for me. I've already experienced it once, so I know that much.
+"Of course I don't...
+ There's nothing waiting for you if you die.
+ I've already experienced it once, so I know that much for a fact.
 }
 [sha:cd2c01c649b99f5e4b705d5b210bb132b2d3d5f1]{
 -- Page 414, Offset 34468.
 -- ……本音を言えば生きていたい。けど、ただ生きているのは、もうダメらしくて」
-...To be honest, I want to live. But... even living on seems hopeless right now."
+To be honest, I want to live.
+ It's just that... I'd have nothing left worth living for."
 }
 [sha:fe95392cb3c5305c8ab0fd3b476810ceddace311]{
 -- Page 415, Offset 34469.
 -- ……そう、ダメだ。
-...That's right. It's hopeless.
+Yeah... there'd be nothing left.
```

Comment:
It's "done for" in 13_3, but that seems to be the only use earlier in the day. 
I changed it because this strikes me as a clearer intent with the context of
> けど、そんな事より。そこにはもう先輩がいない。

Reply:
Well, it's 駄目 earlier, that's why.
He says this stuff a lot. It's his code for "I'm fucked", I guess.

俺はダメだ or 俺は駄目だ.
Anyway... Offset 31650 & 31651 & 33436(+33397) were the ones which are "hope-coded".
The first two are kinda special, and there it was "no hope" because of tricky wordplay. We should probably just leave that one be. But 33436+ should probably be "I'm done for" (the way he speaks to Ciel on the phone). I think it was rendered that way because of 31650, but I suppose we'll just treat that as a special exception.

That said, this line! I think because he goes けど、そんな事より it's meant to sort of contrast with the prior rather than compliment it, if you know what I mean. "Man, me being alive is basically just ダメ (probably here in the meaning of "shouldn't"), I'd kill everyone. But more importantly, no Senpai!"